COMPILATION ERROR: Inlining clone.c:make_clone3_syscall -> multiply defined symbols

**Describe the bug**

This is a general bug, there are several places where labels are used in embedded asms.
This bug concerns the immediate issue of compilation failures when clone.c:make_clone3_syscall is inlined,
causing the label "parent" to be multiply defined.

**To Reproduce**

```
$ clang --version
Debian clang version 16.0.6 (16)
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
$ clang -I ../dynamorio/core/drlibc -I ../dynamorio/core/lib -I ../dynamorio/suite/tests -I . -I include \
  -O1 -c ../dynamorio/suite/tests/linux/clone.c -o clone.o
```

-->

```
In file included from ../dynamorio/suite/tests/linux/clone.c:50:
In file included from ../dynamorio/suite/tests/tools.h:46:
../dynamorio/core/drlibc/drlibc.h:147:43: warning: declaration of
      'struct stat64' will not be visible outside of this function
      [-Wvisibility]
dr_stat_syscall(const char *fname, struct stat64 *st);
                                          ^
../dynamorio/suite/tests/linux/clone.c:253:32: error: symbol 'parent' is
      already defined
                 "call *%%rdx\n\t"
                               ^
<inline asm>:9:2: note: instantiated into assembly here
        parent:
        ^
../dynamorio/suite/tests/linux/clone.c:253:32: error: symbol 'parent' is
      already defined
                 "call *%%rdx\n\t"
                               ^
<inline asm>:9:2: note: instantiated into assembly here
        parent:
        ^
../dynamorio/suite/tests/linux/clone.c:253:32: error: symbol 'parent' is
      already defined
                 "call *%%rdx\n\t"
                               ^
<inline asm>:9:2: note: instantiated into assembly here
        parent:
        ^
../dynamorio/suite/tests/linux/clone.c:253:32: error: symbol 'parent' is
      already defined
                 "call *%%rdx\n\t"
                               ^
<inline asm>:9:2: note: instantiated into assembly here
        parent:
        ^
1 warning and 4 errors generated.
```

**Versions**
 - What version of DynamoRIO are you using? HEAD @ 3da572fb4fdb7021d58c88c421ba5993b9c9c0b9
- What operating system version are you running on? 64 bit linux

--- a/suite/tests/linux/clone.c
+++ b/suite/tests/linux/clone.c
@@ -249,9 +249,9 @@ make_clone3_syscall(void *clone_args, ulong clone_args_size, void (*fcn)(void))
                  "mov %[fcn], %%rdx\n\t"
                  "syscall\n\t"
                  "test %%rax, %%rax\n\t"
-                 "jnz parent\n\t"
+                 "jnz 1f\n\t"
                  "call *%%rdx\n\t"
-                 "parent:\n\t"
+                 "1:\n\t"
                  "mov %%rax, %[result]\n\t"
                  : [ result ] "=m"(result)
                  : [ sys_clone3 ] "i"(CLONE3_SYSCALL_NUM), [ clone_args ] "m"(clone_args),
@@ -265,9 +265,9 @@ make_clone3_syscall(void *clone_args, ulong clone_args_size, void (*fcn)(void))
                  "mov %[fcn], %%edx\n\t"
                  "int $0x80\n\t"
                  "test %%eax, %%eax\n\t"
-                 "jnz parent\n\t"
+                 "jnz 1f\n\t"
                  "call *%%edx\n\t"
-                 "parent:\n\t"
+                 "1:\n\t"
                  "mov %%eax, %[result]\n\t"
                  : [ result ] "=m"(result)
                  : [ sys_clone3 ] "i"(CLONE3_SYSCALL_NUM), [ clone_args ] "m"(clone_args),
@@ -280,9 +280,9 @@ make_clone3_syscall(void *clone_args, ulong clone_args_size, void (*fcn)(void))
                  "ldr x1, %[clone_args_size]\n\t"
                  "ldr x2, %[fcn]\n\t"
                  "svc #0\n\t"
-                 "cbnz x0, parent\n\t"
+                 "cbnz x0, 1f\n\t"
                  "blr x2\n\t"
-                 "parent:\n\t"
+                 "1:\n\t"
                  "str x0, %[result]\n\t"
                  : [ result ] "=m"(result)
                  : [ sys_clone3 ] "i"(CLONE3_SYSCALL_NUM), [ clone_args ] "m"(clone_args),

